about.css

```diff
@@ -0,0 +1,953 @@
+  font-family: 'cocogoose', sans-serif;
+}
+
+#header-section h1 .h1-sub {
+  margin-top: -12px;
+}
+
+#header-section p {
+  border: 3px solid #fff;
+  background-color: #faf7f7;
+  font-size: 0.8rem;
+  font-family: lato, sans-serif;
+  font-weight: 500;
+  padding: 10px;
+  line-height: 1.5;
+  margin: 20px 0 10px 0;
+}
+
+#buy-tickets {
+  display: block;
+  margin-top: 10px;
+  font-size: 1.5rem;
+  font-family: cocogoose, sans-serif;
+}
+
+#date {
+  display: block;
+  margin-top: -8px;
+  font-size: 1.5rem;
+  font-family: cocogoose, sans-serif;
+}
+
+#location {
+  display: block;
+  font-family: lato, sans-serif;
+  font-size: 0.8rem;
+  margin-top: -3px;
+}
+
+#partners {
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  background-color: #434344;
+  padding: 30px 10px;
+}
+
+#partners h2 {
+  color: #fff;
+}
+
+#partners .division-line {
+  width: 50px;
+  height: 2px;
+  background-color: #ec5242;
+  opacity: 0.6;
+  margin: 15px 0 30px 0;
+}
+
+#partners-wrap {
+  display: -webkit-box;
+  display: flex;
+  -ms-flex-wrap: wrap;
+  flex-wrap: wrap;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  gap: 20px;
+}
+
+.logo-container span {
+  color: #ec5242;
+}
+
+.logo-container div {
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+}
+
+.red-div span {
+  font-family: cocogoose, sans-serif;
+}
+
+.logo-container div span {
+  color: black;
+  font-family: cocogoose, sans-serif;
+}
+
+#partners-wrap span {
+  color: #979494;
+  font-size: 1.2rem;
+  background-color: rgba(82, 81, 81, 0.7);
+  border-radius: 6px;
+}
+
+#logo-names a span {
+  font-size: 24px;
+}
+
+.bottom-nav #nav-logo span {
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  margin-left: 5px;
+}
+
+#partners-wrap span:hover {
+  color: #ec5242;
+  cursor: pointer;
+}
+
+footer {
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  padding: 25px 30px;
+  width: 80%;
+  margin-left: 100px;
+}
+
+footer .logo {
+  display: -webkit-box;
+  display: flex;
+  color: black;
+}
+
+footer .logo div {
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  font-size: 0.8rem;
+  font-family: cocogoose, sans-serif;
+  margin-left: 3px;
+}
+
+#footer-p {
+  font-size: 1rem;
+  width: 35%;
+  padding-top: 0;
+  margin-top: 0;
+}
```

Comment:
- [ ] The footer on the about page suppose to have a dark background like the given design. So please fix that.
- [ ] please add the correct height to the footer on both the main page and about page and kindly center the elements inside it too, and try to stick as much as possible to the given Given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/169355303-89fc63c9-2e64-4ff8-a761-a0c0ea09a53c.png)  | ![image](https://user-images.githubusercontent.com/68796232/169355270-d4412e25-ab6b-4305-ba62-dbb520937ee0.png)  |

Reply:
Thank you. I adjusted the height, gave it a dark background, and centered the contents.

![image](https://user-images.githubusercontent.com/59999191/169399050-eb1ba8e5-1682-4054-9c98-11ff6a042d74.png)